Makefile.periph

```diff
@@ -0,0 +1,10 @@
+MODULE ?= periph
+
+# get a list with all periph drivers, that are implemented for the CPU
+HAS = $(wildcard *.c)
+# get a list with the peripherals, that are configured by the board
+WANT = $(patsubst periph_%,%.c,$(filter periph_%,$(FEATURES_PROVIDED)))
+# only compile the peripheral drivers, that are present and configured
+SRC += $(filter $(HAS),$(WANT))
```

Comment:
This breaks if SRC ends up empty, as Makefile.include falls back to *.c in that case.
That needs fixing there, e.g., enclose the SRC lines with ```ifneq(0, AUTO_SRC) ... ```.